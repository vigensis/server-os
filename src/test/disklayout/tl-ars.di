SCSI	c0t5000CCA02203E181d0	HITACHI	HUC109060CSS600	600127266816	no	no
SCSI	c1t5000CCA022040715d0	HITACHI	HUC109060CSS600	600127266816	no	no
SCSI	c2t5000CCA022015A91d0	HITACHI	HUC109060CSS600	600127266816	no	no
SCSI	c3t5000CCA02203D151d0	HITACHI	HUC109060CSS600	600127266816	no	no
SCSI	c4t5000CCA02203E2D1d0	HITACHI	HUC109060CSS600	600127266816	no	no
SCSI	c12t5000CCA0490428ADd0	HGST	HUSMH8010BSS204	100030242816	no	yes
USB	c16t0d0	Kingston	DTR30G2	15737028608	yes	no
