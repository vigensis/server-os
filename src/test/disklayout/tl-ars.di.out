{
	"vdevs": [
		{
			"type": "raidz2",
			"devices": [
				{
					"name": "c0t5000CCA02203E181d0",
					"vid": "HITACHI",
					"pid": "HUC109060CSS600",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "c1t5000CCA022040715d0",
					"vid": "HITACHI",
					"pid": "HUC109060CSS600",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "c2t5000CCA022015A91d0",
					"vid": "HITACHI",
					"pid": "HUC109060CSS600",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "c3t5000CCA02203D151d0",
					"vid": "HITACHI",
					"pid": "HUC109060CSS600",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "c4t5000CCA02203E2D1d0",
					"vid": "HITACHI",
					"pid": "HUC109060CSS600",
					"size": "600127266816",
					"solid_state": false
				}
			]
		}
	],
	"capacity": 1800381800448,
	"logs": [
		{
			"name": "c12t5000CCA0490428ADd0",
			"vid": "HGST",
			"pid": "HUSMH8010BSS204",
			"size": "100030242816",
			"solid_state": true
		}
	]
}
